ppr-ui/src/composables/useRegistration.ts

```diff
@@ -206,15 +206,17 @@ export const useRegistration = () => {
       if (localState.submittedStartDate && localState.submittedEndDate) {
         const sDateFilter = (item: any): boolean => {
           if (item.createDateTime) {
-            const created = item.createDateTime.substring(0, 10)
-            if (created < localState.submittedStartDate) return false
+            const createdDate = new Date(item.createDateTime)
+            const localDateStr = createdDate.toLocaleDateString('en-CA')
```

Comment:
Do we want to set the time zone on this?

Reply:
good question, let me look

Reply:
alright so I actually took out the timezone in the datepicker so that it also displays for whatever the users local timezone is. This is because that's also how it displays in the table and we don't show the timezone anywhere for now (we may come back to this if the ux team wants to though)